tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -195,77 +188,69 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+            fluid.tests.getTooltipContents(linksList, pageSize));
+        fluid.tests.verifyToolTipsOfPagerBar(pager, "bottom", pageSize,
+            fluid.tests.getTooltipContents(linksList, pageSize));
     };
 
-    /**
-     * verify whether the displaying links and their tooltips
-     */
-    fluid.tests.verifyToolTips = function (pager, sequence, pageSize, expectedTooltips) {
+    fluid.tests.getPagerBar = function (pager, location) {
+        return pager[location === "top" ? "pagerBar" : "pagerBar-1"];
+    };
+
+    fluid.tests.verifyPagerBar = function (pager, location, pageSize) {
         var expectedNumberOfLinks = Math.ceil(pager.dataModel.length / pageSize);
-        sequence.push({
-            func: function () {
-                pager.locate("pageSize").val(pageSize);
-                jqUnit.assertEquals("When the page size is " + pager.model.pageSize + ", Top pageLinks",
-                    expectedNumberOfLinks,
-                    $("#rendered-ioc .flc-pager-top .flc-pager-links .flc-pager-pageLink a").length);
-                jqUnit.assertEquals("When the page size is " + pager.model.pageSize + ", Bottom pageLinks",
-                    expectedNumberOfLinks,
-                    $("#rendered-ioc .flc-pager-bottom .flc-pager-links .flc-pager-pageLink a").length);
-            }
+
+        //Verify the number of links in the pager bar
+        jqUnit.assertEquals("The number of links in the pager should be according to the page size",
+            expectedNumberOfLinks,
+            fluid.tests.getPagerBar(pager, location).locate("pageLinks").length);
+    };
+
+    fluid.tests.assertVisibleTips = function (pager, message, targetIds, expectedTooltip) {
+        fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
+            jqUnit.assertNode(tooltip.length, expectedTooltip, $("b", tooltip));
         });
+    };
 
+    fluid.tests.verifyToolTipsOfPagerBar = function (pager, location, pageSize, expectedTooltips) {
+        var expectedNumberOfLinks = Math.ceil(pager.dataModel.length / pageSize);
         for (var x = 0; x < expectedNumberOfLinks; x++) {
-            fluid.tests.verifyToolTip(pager, sequence, "top", x, expectedTooltips[x]);
-            fluid.tests.verifyToolTip(pager, sequence, "bottom", x, expectedTooltips[x]);
+            fluid.tests.verifyPagerToolTip(pager, location, x, expectedTooltips[x]);
         }
     };
 
-    /**
-     * Verify whether the tooltip of the given index is matched with the provided tooltip
-     */
-    fluid.tests.verifyToolTip = function (pager, sequence, location, index, expectedTooltip) {
-        var link =  "#rendered-ioc .flc-pager-" + location + " .flc-pager-links .flc-pager-pageLink a:eq(" + index + ")";
+    fluid.tests.verifyPagerToolTip = function (pager, location, index, expectedTooltip) {
+        var link =  $(fluid.tests.getPagerBar(pager, location).locate("pageLinks").find("a").eq(index));
```

Comment:
you don't need to wrap this in a jQuery `$()` it's already a jQuery element because of using `eq(index)`